src/Components/ActivationKeys/index.js

```diff
@@ -1,31 +1,36 @@
 import React from 'react';
+import { TextVariants } from '@patternfly/react-core';
+import { Main } from '@redhat-cloud-services/frontend-components/Main';
 import {
-  Bullseye,
-  Button,
-  EmptyState,
-  EmptyStateIcon,
-  Title,
-} from '@patternfly/react-core';
-import { LockIcon } from '@patternfly/react-icons';
+  PageHeader,
+  PageHeaderTitle,
+} from '@redhat-cloud-services/frontend-components/PageHeader';
+import ActivationKeysTable from '../ActivationKeysTable';
+import { useQueryClient } from 'react-query';
+import NoAccessView from './no-access';
 
-const ActivationKeys = () => (
-  <Bullseye>
-    <EmptyState>
-      <EmptyStateIcon icon={LockIcon} />
-      <Title headingLevel="h4" size="lg">
-        This feature is not currently supported
-      </Title>
-      <Button
-        component="a"
-        variant="link"
-        href="https://access.redhat.com/articles/simple-content-access"
-        target="_blank"
-        rel="noreferrer noopener"
-      >
-        Customer portal
-      </Button>
-    </EmptyState>
-  </Bullseye>
-);
+const ActivationKeys = () => {
+  const queryClient = useQueryClient();
+  const user = queryClient.getQueryData('user');
+  const Page = () => {
+    return (
+      <React.Fragment>
+        <PageHeader>
+          <PageHeaderTitle title="Activation Keys" />
+          <TextVariants.p>Orginazaion ID: {user.orgId}</TextVariants.p>
```

Comment:
`TextVariants.p` is just supposed to be passed as a prop to the Text component, which is supposed to be nested in a TextContent component.  The patternfly docs for it are [here](https://www.patternfly.org/v4/components/text).